documents/2-RTB/Documentazione_Esterna/Glossario.typ

```diff
@@ -0,0 +1,425 @@
+prodotto e può includere ricercatori, architetti, designer, programmatori, e molti altri
+
+== #LTG("Diagramma dei casi d'uso")
+
+Un diagramma di casi d'uso è uno strumento grafico utilizzato nella modellazione UML (Unified Modeling Language) per
+rappresentare le interazioni tra gli attori (utenti o sistemi esterni) e un sistema software. Serve a descrivere le
+funzionalità del sistema dal punto di vista dell'utente e a documentare i requisiti funzionali in modo chiaro e visivo.
+
+#pagebreak()
+
+= #LTG("E")
+#pagebreak()
+
+= #LTG("F")
+#pagebreak()
+
+= #LTG("G")
+
+== #LTG("GitHub")
+
+GitHub è una piattaforma di sviluppo collaborativo basata su Git, un sistema di controllo delle versioni distribuito. È
+ampiamente utilizzato da sviluppatori di software e team di sviluppo per gestire progetti, tenere traccia delle
+modifiche al codice sorgente, coordinare il lavoro tra i membri del team e facilitare la collaborazione su larga scala
+
+== #LTG("Glossario")
+
+Elenco strutturato di termini tecnici o specializzati, ognuno corredato dalla propria definizione o spiegazione. Questo
+strumento aiuta a migliorare la comunicazione tra le varie parti coinvolte in un progetto, riducendo le ambiguità e
+garantendo una comprensione condivisa dei termini utilizzati in un determinato contesto
+
+== #LTG("GUI (Graphical User Interface)")
+
+Una GUI (acronimo di Graphical User Interface, in italiano Interfaccia Grafica Utente) è un'interfaccia che consente
+agli utenti di interagire con un sistema informatico o un software attraverso elementi visivi e grafici, anziché
+utilizzare solo comandi testuali o linee di comando.
+
+#pagebreak()
+
+= #LTG("H")
+#pagebreak()
+
+= #LTG("I")
+#pagebreak()
+
+= #LTG("J")
+
+== #LTG("Jira")
+
+Jira è uno strumento software sviluppato da Atlassian, utilizzato per la gestione di progetti, il monitoraggio dei
+problemi e il controllo dello sviluppo agile. È particolarmente popolare tra i team di sviluppo software, ma è anche
+ampiamente utilizzato in altri ambiti aziendali.
+
+#pagebreak()
+
+= #LTG("K")
+
+== #LTG("Key Performance Indicator (KPI)")
+
+I Key Performance Indicator (KPI) sono misure quantitative utilizzate per valutare le prestazioni di un'organizzazione,
+di un progetto o di un processo rispetto agli obiettivi prefissati. I KPI forniscono un modo oggettivo per monitorare e
+valutare il successo di un'attività, identificare le aree di miglioramento e prendere decisioni informate per
+ottimizzare le prestazioni
+
+#pagebreak()
+
+= #LTG("L")
+
+== #LTG("LaTeX")
+
+LaTeX è un linguaggio di markup e un sistema di preparazione di documenti utilizzato principalmente per la produzione di
+documenti scientifici, accademici e tecnici, ma anche per altri tipi di pubblicazioni come libri, tesi, articoli e
+presentazioni. È basato su TeX, un sistema di tipografia creato da Donald Knuth negli anni '70, ma LaTeX aggiunge una
```

Comment:
non so come mai, ma in Glossary.pdf quel " '70 ", compare come "70. Sul sito invece compare normalmente con '70. Riesci a darci un occhio? In caso con un bel 1970 si risolve tutto

Reply:
done, ma molto strano. Typst compila il ' come " se non c'è una parola davanti. da ricontrollare